nisystemlink/clients/product/models/_product_response.py

```diff
@@ -0,0 +1,51 @@
+from datetime import datetime
+from typing import Dict, List, Optional
+
+from nisystemlink.clients.core._uplink._json_model import JsonModel
+
+
+class ProductResponse(JsonModel):
```

Comment:
I don't like this rename. The core response record should just be `Product`. All the fields should be Optional. It should be used for both the create response and the query response.